### PR TITLE
Update type sigs to conform to Haskell notation

### PR DESCRIPTION
**Pull request by svozza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,4 +281,4 @@ Thanks to the team behind [Ramda.js](https://github.com/ramda/ramda) and [moment
 ## Contributing
 Yes please!
 
-Something to note is that automatically we release to NPM on every successful commit and we also strictly follow SemVer so please update the version number in `package.json` appropriately when submitting a Pull Request.
+Something to note is that we  automatically release to NPM on every successful commit and we also strictly follow SemVer so please update the version number in `package.json` appropriately when submitting a Pull Request.
```
Comment (davidchambers) [inline README.md]: You have two spaces after "we".

File: README.md [hunk 2]
```diff
@@ -45,8 +44,7 @@ tomorrowAsString(new Date('2015-01-01')); // '2015-01-02';
 
 ## API
 
-### parse
-`String -> Date`
+### parse :: String -> Date
```
Comment (davidchambers) [inline README.md]: It's worth noting that you'll lose nice anchors. This'll become [#parse--string---date](#parse--string---date), for example. [Transcribe](https://github.com/plaid/transcribe) provides a partial solution to this problem.

On a different note, I suggest wrapping the heading in backticks:

``` diff
-### parse :: String -> Date
+### `parse :: String -> Date`
```

Comment (cullophid) [inline README.md]: that is a valid point. im looking at putting the docs in a gitbook wich would change all of this

Comment (svozza) [inline README.md]: My main thinking behind this was to make the docs for BF and date-fp to be consistent.

**Comment by svozza** [comment]
Like the Gitbooks! Superseded by #25.

**Pull request closed without merging**